package.json

```diff
@@ -38,17 +38,17 @@
     }
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
```

Comment:
I'd hold of on using the --hoist flag until we can verify it works with the theme repos. Right now, I know it works with terra-ui, but I wasn't able to get it working the cerner-consumer-theme repo.

Reply:
Sounds good, but we should pause this PR until we can verify `--hoist` works. It significantly lowers the memory footprint of the dev site.